### DISCUSSION TITLE
HD playback?

### DISCUSSION THREAD

**Discussion by jmmontoro** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#6] [General]
Hey, this works, and that is a huge progress since nothing worked anymore. But I noticed it doesn't go higher than 480p, is that how it's supposed to work?

**Comment by aidanmacgregor** [comment]
Hey, So unfortunately without Widevine L1 its not possible to decrypt/decode HD video streams sent by Netflix servers, There was HD mods for a short while but they always required adding your own FULL esn (as reported by the Kodi add on) and often some experimentation was still needed :)

**Reply by jmmontoro** [reply]
Ah yes, I tried swapping out my ESN but the app didn't even connect.
